assets/admin/editor-wizard/steps/course-upgrade-step.js

```diff
@@ -98,10 +99,34 @@ const FeatureItem = ( { children } ) => (
 	</li>
 );
 
+/**
+ * Get the upgrade URL based on the environment.
+ *
+ * @param {string}  siteSlug       The site slug.
+ * @param {boolean} isWpcomHosting Whether the site is on WPCOM.
+ *
+ * @return {string} The upgrade URL.
+ */
+const getUpgradeUrl = ( siteSlug, isWpcomHosting ) => {
+	const sensieParams = new URLSearchParams( {
```

Comment:
```suggestion
	const senseiParams = new URLSearchParams( {
```

And we also need to rename where we're using the variable.